### DISCUSSION TITLE
Configuration FRW

### DISCUSSION THREAD

**Discussion by maxpouliot** [MTESSDev/FRW#4] [Annonces]
Dans notre bac à sable, nous avons le message : This extension has been disabled because the current workspace is not trusted. Notez que nous avons pas ouvert le bac à sable depuis quelques mois. 
Merci pour l'aide!

**Comment by anisite** [comment]
C'est probablement en lien avec la sécurité du workspace?

Voir si ça peut vous aider:

https://code.visualstudio.com/docs/editor/workspace-trust

**Reply by maxpouliot** [reply]
Yes merci. On a trouvé!
